crates/sncast/tests/e2e/account/create.rs

```diff
@@ -49,11 +49,10 @@ pub async fn test_happy_case(account_type: &str) {
         indoc! {r"
         command: account create
         add_profile: --add-profile flag was not set. No profile added to snfoundry.toml
-        address: 0x0[..]
-        max_fee: [..]
-        message: Account successfully created. Prefund generated address with at least <max_fee> STRK tokens. It is good to send more in the case of higher demand.
+        account_address: 0x0[..]
+        message: Account successfully created. The estimated deployment fee is [..]
```

Comment:
Why aren't we asserting the whole message. We should have at least one tests that does that.